test/integration/spec/connect.js

```diff
@@ -203,6 +204,119 @@ describe('connect', function() {
     });
   });
 
+  describe.only('media region', () => {
+    let sid = null;
+    let token = null;
+    let roomName = null;
+
+    beforeEach(async () => {
+      const identity = randomName();
+      token = getToken(identity);
+      sid = await createRoom(randomName(), defaults.topology, { MediaRegion: 'gll' });
+    });
+
+    const invalidRegions = ['foo', '34', '$abc', 'abc!'];
+    const validRegions = defaults.regions ? defaults.regions.split(',') : ['au1', 'br1', 'de1', 'ie1', 'in1', 'jp1', 'sg1', 'us1', 'us2'];
```

Comment:
Are these tests ever intended to be run in other environments than prod? The regions listed are valid in prod, but not in dev or stage which support a subset of this list.

Reply:
@ceaglest We take in an environment variable `REGIONS=rg1,rg2,...` to override the list of regions in non-prod environments.